src/main/cms/cms_menu_gps_lap_timer.c

```diff
@@ -0,0 +1,128 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
Do we need updated license?

Reply:
See [src/main/fc/gps_lap_timer.h](https://github.com/betaflight/betaflight/pull/11856/files#diff-acfa8e57f55d7effc8d629851e13d0c8ca7108aba9ef09df4d1bcd3988e7095e)

Reply:
Good call, forgot to mention it